api/scripts/database_size_report.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+#   Runs a database table size query for all tables. 
+#   Query is from: https://wiki.postgresql.org/wiki/Disk_Usage (General Table Size Information)
+#
+# Usage:
+#
+#   PROJECT=OPENSHIFT_PROJECT_NAME POD=OPENSHIFT_POD DATABASE=POSTGRESQL_DATABASE ${THIS_FILE}
+#
+# Example:
+#
+#   e.g. PROJECT=e1e498-prod POD=patroni-wps-prod-2 DATABASE=wps ./database_size_report.sh
+
+set -euo pipefail
+
+SCRIPT_DIR="$(dirname "$0")"
+
+source "$SCRIPT_DIR/common_oc_checks.sh"
+
+# From https://wiki.postgresql.org/wiki/Disk_Usage
+SIZE_REPORT_QUERY="SELECT *, pg_size_pretty(total_bytes) AS total
+    , pg_size_pretty(index_bytes) AS index
+    , pg_size_pretty(toast_bytes) AS toast
```

Comment:
what are toast bytes?

Reply:
Postgres does not support physical row data that crosses the size of a data block. To enable values larger than data blocks it breaks them up into smaller pieces and are stored in the TOAST table that is related to the original table.

https://wiki.postgresql.org/wiki/TOAST

Reply:
@conbrad could you add that comment into the script please?